service/s3/s3manager/upload.go

```diff
@@ -373,15 +379,15 @@ func (u *uploader) upload() (*UploadOutput, error) {
 	}
 
 	// Do one read to determine if we have more than one part
-	reader, _, part, err := u.nextReader()
+	reader, _, part, callback, err := u.nextReader()
 	if err == io.EOF { // single part
 		return u.singlePart(reader)
 	} else if err != nil {
 		return nil, awserr.New("ReadRequestBody", "read upload data failed", err)
 	}
 
 	mu := multiuploader{uploader: u}
-	return mu.upload(reader, part)
+	return mu.upload(reader, part, callback)
```

Comment:
what do you think about renaming `callback` to be `cleanup` similar to the other usages of this value?

Reply:
Missed this one when I went through a renamed from callback => cleanup.